config.example.json

```diff
@@ -5,5 +5,6 @@
     "port": "6379"
   },
   "port": 3000,
-  "interval": 600000
-}
+  "interval": 600000,
+  "nyt_interval": 72e6
```

Comment:
72e6 milliseconds is like 1200 minutes i.e updates every 1200 minutes (20 hours.) Are you sure this is the value you want?

Reply:
They only update their data once a day

Reply:
After talking on Discord, changed the interval to 1 hour.